### DISCUSSION TITLE
It's hard to unfollow users with private profiles

### DISCUSSION THREAD

**Discussion by Alan-Liang** [community/community#15593] [Other Feature Feedback, Questions, & Ideas]
When a user hides its profile, GitHub hides not only its Follow button, but also its Unfollow button, making it hard to unfollow users with private profiles. Furthermore, if you are following such a user, its activity is still shown on the dashboard page (i.e. home page). Is that by design?

**Comment by ZenithClown** [comment]
GitHub [documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/setting-your-profile-to-private) says:
> By making your profile private, you will not remove or hide past activity; this setting only applies to your activity while the private setting is enabled.
